.github/workflows/docs.yml

```
@@ -8,15 +8,20 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@master
```

```suggestion
      uses: actions/checkout@v2
```
I think for Github Actions workflows, we generally want to use fixed versions rather than latest in case of breaking changes being merged into the Action which could break our CI. What do you think?